src/Buildalyzer/Construction/PackageReference.cs

```diff
@@ -10,6 +10,8 @@ public class PackageReference : IPackageReference
     internal PackageReference(XElement packageReferenceElement)
     {
         this.Name = packageReferenceElement.GetAttributeValue("Include") ?? packageReferenceElement.GetAttributeValue("Update");
-        this.Version = packageReferenceElement.GetAttributeValue("Version");
+        var versionElement = packageReferenceElement.DescendantsAndSelf()
+            .FirstOrDefault(x => x.Name.LocalName == "Version");
+        this.Version = versionElement?.Value ?? packageReferenceElement.GetAttributeValue("Version");

```

Comment:
Are you sure that the `<Version>x</Version>` overrules the `Version` attribute?

Reply:
It feels weird (and wrong) for sure, but it should pick the same one (if they differ) as MS Build does. (and in case MS Build does return an error we might want to document that).